//Try Catch --Error Handling Method-----------------

let a = 10;
let b = 30;

console.log("a =", a);
console.log("b =", b);
console.log("a+b =", a + b);
try {
  console.log("a+c =", a + c);
} catch (err) {
  console.log(err);
}
console.log("a*b =", a * b);
